src/magicbox/MagicBox.ts

```diff
@@ -235,11 +234,10 @@ export class MagicBoxInstance {
     this.inputManager.blur();
   }
 
-  public clearSuggestion() {
-    this.suggestionsManager.mergeSuggestions([], suggestions => {
-      this.updateSuggestion(suggestions);
-    });
+  public async clearSuggestion() {
     this.inputManager.setWordCompletion(null);
+    const suggestions = await this.suggestionsManager.mergeSuggestions([]);
+    this.updateSuggestion(suggestions);
```

Comment:
I am curious what passing an empty array to `this.suggestionsManager.mergeSuggestions` does?

Also, are we storing suggestions inside of the `SuggestionsManager` instance as well as this MagicBox class? If so, it would better to keep one source of truth, likely inside the `SuggestionsManager`, to avoid bugs where we forget to update the local copy inside `MagicBox`.